pandas/core/generic.py

```diff
@@ -6502,7 +6502,7 @@ def replace(self, to_replace=None, value=None, inplace=False, limit=None,
                 regex = True
 
             items = list(to_replace.items())
-            keys, values = lzip(*items) or ([], [])
+            keys, values = zip(*items) if len(items) else ([], [])
```

Comment:
since `items` is a `list` I think you can modify the ternary to be `zip(*items) if items else ([], [])`